test/Test_SpeedToPath_RegularStepGradientDescent_3D.cxx

```diff
@@ -1,3 +1,33 @@
+/*=========================================================================
```

Comment:
This license header is out of date.  Please use the header here:

  https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/include/itkConstSliceIterator.h#L1-L17